src/app/components/experiment-workspace/experiment-workspace.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="workspace-content">
-    <qhana-plugin-sidebar class="sidebar"></qhana-plugin-sidebar>
+    <qhana-plugin-sidebar class="sidebar" (tabSelected)="editTemplateTab($event)"></qhana-plugin-sidebar>
```

Comment:
Communication about what is selected here should preferably happen using query parameters.